SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -888,7 +901,24 @@ void USpatialSender::SendCreateEntityRequest(USpatialActorChannel* Channel)
 
 void USpatialSender::SendDeleteEntityRequest(Worker_EntityId EntityId)
 {
-	Connection->SendDeleteEntityRequest(EntityId);
+	const AActor* Actor = Cast<AActor>(PackageMap->GetObjectFromEntityId(EntityId).Get());
+
+	if (Actor && Actor->IsNetStartupActor() && Actor->GetIsReplicated())
```

Comment:
checking the IsReplicated flag is likely unnecessary/implicit, since we wouldn't be trying to delete an entity otherwise